examples/browser/wdio.conf.js

```diff
@@ -5,9 +5,14 @@ const path = require('path');
 const port = 3000;
 const host = 'localhost';
 
+const debug = process.env.DEBUG;
```

Comment:
does not seem to be used anywhere

Reply:
It is used below for the capabilities

Reply:
It should be passed to config as well: https://github.com/theia-ide/theia/pull/611/files#r154377323